src/ast/ast.h

```diff
@@ -1685,12 +1685,16 @@ class CallBase : public Expression {
     return SpreadPositionField::decode(bit_field_);
   }
 
+  int lparen_token_position() const { return lparen_token_position_; }
```

Comment:
TODO: we should rename this since we want to also use this for the position of the first backtick in tagged template calls like:
```ts
call`foo`
```